CactusTutorial.ipynb

```diff
@@ -162,7 +195,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!./simfactory/bin/sim build --mdbkey make 'make -j1' --thornlist ../einsteintoolkit.th | cat"
+    "%%bash\n",
+    "./simfactory/bin/sim build --mdbkey make 'make -j2' --thornlist ../einsteintoolkit.th"
```

Comment:
We need to check if the memory issue on the NDS machine has been addressed.

Reply:
I ran `rm -rf configs/sim ; simfactory/bin/sim build --mdbkey make 'make -j2' --thornlist ../einsteintoolkit.th` in an infinite loop over the weekend. 

Other than making the uptime monitor think that the nds host was offline / online every other minute this seems to not have had any detrimentous effect. I have  limited the VM memory to 4GB which is 1/2 of the physical memory of the host (before it was 8GB which is the physical amount of memory on the host, which made it possible for the container to crowd out eg the outside webserver since there cannot be swap on this host).